### ISSUE TITLE
Add example in request body json

### ISSUE THREAD

**Issue by pei0804** [swaggo/swag#55]
For example.

```go
type Bottle struct {
	ID        int    `json:"id"`
	Name      string `json:"name"`
	AccountID int    `json:"account_id"`
}
```

```go
// GetExample get method example
// @Summary Add a new pet to the store
// @Description get string by ID
// @ID id
// @Accept json
// @Produce json
// @Param bottles body model.Bottle true "bottle info"
// @Success 200 {object} model.Bottle "bottle info" <--- 
// @Router /bottles [post]
func PostBottles(c echo.Context) error {
	b := model.Bottle{
		ID:        1,
		Name:      "Bottle",
		AccountID: 1,
	}
	return c.JSON(http.StatusOK, b)
}
```

Display json example in swagger.

```go
{
  "account_id": 0,
  "id": 0,
  "name": "string"
}
```

I want to real example.
https://swagger.io/docs/specification/2-0/adding-examples/

```go
{
  "account_id": 1,
  "id": 2,
  "name": "red wine No14"
}
```

What kind of plan are you planning to do in the future?

---

By the way, I thought approach. 

```go
type Bottle struct {
	ID        int    `json:"id" example:"2"`
	Name      string `json:"name" example:"red wine No14"`
	AccountID int    `json:"account_id" example:"1"`
}
```

**Comment by easonlin404** [comment]
I think that using `example` tag it's  a good approach for Property Examples and Object Examples.

**Comment by pei0804** [comment]
OK.
If i have time. I try it.

**Comment by easonlin404** [comment]
fixed by #56

**Comment by huahuayu** [comment]
@pei0804 @easonlin404 for `POST` it works, but for `GET` query parameters, how cam I give field example?
